pkg/agent/controller/agent.go

```diff
@@ -78,24 +78,32 @@ func (a *Controller) Run(stopCh <-chan struct{}) error {
 	return nil
 }
 
-func (a *Controller) serviceExportToRemoteMcs(obj runtime.Object) runtime.Object {
+func (a *Controller) serviceExportToRemoteMcs(obj runtime.Object) (runtime.Object, bool) {
 	svcExport := obj.(*lighthousev2a1.ServiceExport)
 	svc, err := a.kubeClientSet.CoreV1().Services(svcExport.Namespace).Get(svcExport.Name, metav1.GetOptions{})
 	if err != nil {
 		klog.Errorf("No matching service for %v", svcExport)
-		return nil
+		return nil, true
 	}
-	mcs := &lighthousev1.MultiClusterService{
+
+	serviceImport := &lighthousev2a1.ServiceImport{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: svc.Name + "-" + svc.Namespace + "-" + a.clusterID,
 			Annotations: map[string]string{
 				"origin-name":      svc.Name,
 				"origin-namespace": svc.Namespace,
 			},
 		},
-		Spec: lighthousev1.MultiClusterServiceSpec{
-			Items: []lighthousev1.ClusterServiceInfo{
-				a.newClusterServiceInfo(svc, a.clusterID),
+		Spec: lighthousev2a1.ServiceImportSpec{
+			Ports:                 nil,
+			IP:                    "",
+			Type:                  lighthousev2a1.SuperclusterIP,
+			SessionAffinity:       "",
+			SessionAffinityConfig: nil,
```

Comment:
It's not necessary to explicitly initialize these fields too their nil values as that's done automatically.

Reply:
Done.